tensorpack/tfutils/summary.py

```diff
@@ -258,10 +265,3 @@ def add_moving_summary(*args, **kwargs):
             # TODO a new collection to summary every step?
             tf.add_to_collection(coll, op)
     return ema_ops
-
-
-try:
-    import scipy.misc
-except ImportError:
-    from ..utils.develop import create_dummy_func
-    create_image_summary = create_dummy_func('create_image_summary', 'scipy.misc')  # noqa
```

Comment:
cv2 is an __optional__ dependency. Don't import it at the beginning of file but put it here.